src/Tools/Source/OpenSourceDebug/OpenSourceDebug.csproj

```diff
@@ -31,14 +31,13 @@
     <VSSDKTargetPlatformRegRootSuffix>Roslyn</VSSDKTargetPlatformRegRootSuffix>
     <SolutionDir Condition="'$(SolutionDir)' == '' OR '$(SolutionDir)' == '*Undefined*'">..\..\..\..\</SolutionDir>
     <NonShipping>true</NonShipping>
+    <DeployExtension Condition="'$(DeployExtension)' == ''">True</DeployExtension>
```

Comment:
I don't understand the intent of this change. /p:DeployExtension=false overrides per MSBuild rules without the condition.

(Or don't touch this entirely....this will just cause merge conflicts for me...)